frontend/packages/dev-console/src/components/edit-application/EditApplicationPage.tsx

```diff
@@ -1,99 +1,143 @@
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
-import { Firehose, FirehoseResource, LoadingBox } from '@console/internal/components/utils';
-import { ImageStreamModel } from '@console/internal/models';
 import { RouteComponentProps } from 'react-router-dom';
 import { Helmet } from 'react-helmet';
+import { LoadingBox } from '@console/internal/components/utils';
+import { K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
+import {
+  useK8sWatchResource,
+  useK8sWatchResources,
+  WatchK8sResults,
+  WatchK8sResultsObject,
+} from '@console/internal/components/utils/k8s-watch-hook';
 import { ServiceModel } from '@console/knative-plugin';
-import { referenceForModel } from '@console/internal/module/k8s';
 import { PipelineModel } from '@console/pipelines-plugin/src/models';
+import { PipelineKind } from '@console/pipelines-plugin/src/types';
 import NamespacedPage, { NamespacedPageVariants } from '../NamespacedPage';
-import EditApplication from './EditApplication';
-import { EditApplicationProps } from './edit-application-types';
+import EditApplicationComponent from './EditApplicationComponent';
 
-const INSTANCE_LABEL = 'app.kubernetes.io/instance';
-const EditApplicationComponentLoader: React.FunctionComponent<EditApplicationProps> = (
-  props: EditApplicationProps,
-) => {
-  const { loaded } = props;
-  return loaded ? <EditApplication {...props} /> : <LoadingBox />;
+type watchResource = {
```

Comment:
type should be in pascal case
```suggestion
type WatchResource = {
```